gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
+Establish a marketplace by creating a website(s) that lists InnerSource project needs as "gigs". 
+You can attract potential contributors by posting gigs with explicit requirements (amount of time, skills). People can sign up for a gig and garner approval from their management for those explicit requirements. Individual contributors are then awarded points for the gigs they complete. 
+This makes it explicit what the commitment is and that management agrees with it while encouraging and tracking involvement.
+
+## Problem
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other and connect.
+
+## Story (Optional)
+Your InnerSource program has been established and there is great interest but little actual activity. 
+Individual project owners are encountering difficulties attracting a community of contributors.
+Most managers are supportive in allowing their employees the time to involve themselves in an InnerSource project. 
+There simply is no single uniform way for contributors to discover all the InnerSource opportunities that exist that they might be interested in contributing to.
+Conversely, innerSource project owners also have no way of informing a general audience about their projects and its goals.
+
+## Context
+* There are multiple business units in your organization that do not communicate well.
```

Comment:
from @rshanmer : does the company really have to have multiple BUs? I propose to delete this bulletpoint
from @tim: is the pattern applicable regardless of whether there are multiple BUs?
from @rshanmer : lack of communication seems to be the core here
from @tim: hesitate to remove this completely. There should be some mention of the specific instance
from @rrrutledge : maybe in the known contexts?
from @rshanmer : focus on the structure.
from @stephenmccall : rephrase: "there are multiple organizational units which don't have good visibility into each other". 
from @dicortazar : might this also be not only about communication of visibility but also about units not being _welcoming_ to each other?
from @rshanmer : another idea: there isn't great communication in the companies, e. g. because we're structured in business units (silos). Give some examples that people can relate to.
from @NewMexicoKid : There isn't great communication across the company; e.g., beecause of BU silos or multiple orgs that don't have visibility into each other's activities